File: RedSheet.DbEntities/Models/SayingOpinion.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("SayingOpinions", Schema = "dbo")]
	[Serializable]
	public partial class SayingOpinion  
	{
 
	[Range(1, int.MaxValue)]
	public int CultureCountryId { get; set; } 
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int SayingOpinionId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string SayingOpinionValue { get; set; } 

	public SayingOpinion()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/saying-opinion.ts
import { required, maxLength, range } from '@rx/annotations';

export class SayingOpinion {
    constructor(sayingOpinion?: SayingOpinion )  {
        let properties = [ "cultureCountryId", "sayingOpinionId", "sayingOpinionValue",];
        for (let property of properties)
            if (sayingOpinion && sayingOpinion[property])
                this[property] = sayingOpinion[property];
    }
 
    @range(1,2147483647)
	cultureCountryId : number =   undefined;
 
	sayingOpinionId : number =   0 ;
 
    @required()
    @maxLength(500)
	sayingOpinionValue : string =   undefined;
}
